config/sync/core.entity_view_display.paragraph.stanford_media_caption.default.yml

```diff
@@ -61,9 +61,9 @@ content:
     label: hidden
     settings:
       image:
-        image_formatter: image_style
```

Comment:
Snuck this in here as it really should have been a responsive image to begin with.